content/en/agent/faq/rpm-gpg-key-rotation-agent-6.md

```diff
@@ -45,6 +45,8 @@ package gpg-pubkey-e09422b3 is not installed
 
 ## How to manually trust the new GPG key
```

Comment:
nit: Having "manual" in the title here is a bit misleading. I'd remove `manually` from the title, and re-iterate in an introduction of this section that this step is not needed if users are using a recent version of an official install method, or if their hosts already trust the new key as described in the previous section.